package.json

```diff
@@ -6,10 +6,10 @@
   ],
   "packageManager": "yarn@3.4.1",
   "devDependencies": {
-    "@react-native-community/eslint-config": "3.2.0",
-    "eslint": "8.26.0",
-    "prettier": "2.8.8",
-    "typescript": "4.9.5"
+    "@rnx-kit/eslint-plugin": "^0.5.0",
```

Comment:
why not continue with `@react-native/eslint-config` ?

Reply:
My issue with it is that it's doesn't use a preset. It is completely custom: https://github.com/facebook/react-native/blob/main/packages/eslint-config-react-native/index.js

Here's an example: They recently enabled a rule: https://github.com/facebook/react-native/pull/41272

This rule is enabled by default if you use the recommended preset: https://eslint.org/docs/latest/rules/no-global-assign

`@rnx-kit/eslint-plugin` is basically extending four recommended presets plus some rules for better tree shaking: https://github.com/microsoft/rnx-kit/blob/main/packages/eslint-plugin/src/configs/recommended.js

Reply:
thanks for in-depth take on it 🙏 make more sense now